invenio_vocabularies/resources/resource.py

```diff
@@ -164,3 +135,37 @@ def launch(self):
         """Create a task."""
         self.service.launch(g.identity, resource_requestctx.data or {})
         return "", 202
+
+
+class VocabulariesAdminResource(RecordResource):
+    """Resource for vocabularies admin interface."""
+
+    def create_url_rules(self):
+        """Create the URL rules for the record resource."""
+        routes = self.config.routes
+        rules = super().create_url_rules()
+
+        rules.append(
+            route("GET", routes["all"], self.get_all_vocabulary_types),
+        )
+        return rules
+
+    @request_search_args
+    @response_handler(many=True)
+    def get_all_vocabulary_types(self):
+        """Return information about _all_ vocabularies."""
+        identity = g.identity
+        hits = self.service.search(identity, params=resource_requestctx.args)
+
+        return hits.to_dict(), 200
+
+    @request_view_args
+    @response_handler()
+    def read(self):
+        """Read an item."""
+        pid_value = (
+            resource_requestctx.view_args["type"],
+            resource_requestctx.view_args["pid_value"],
+        )
+        item = self.service.read(g.identity, pid_value)
```

Comment:
minor: not sure if this will work without overriding/defining `VocabularyTypeService.read(...)` to use the parameters to fetch the vocabulary type from the DB.

On the other hand, I'm not sure if this is also used anywhere.

Reply:
How could I check if this works?